pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -861,12 +861,14 @@ func validateNetworkHasOnlyOneType(field *k8sfield.Path, cniTypesCount int, caus
 func validateBootOrder(field *k8sfield.Path, spec *v1.VirtualMachineInstanceSpec, volumeNameMap map[string]*v1.Volume) (bootOrderMap map[uint]bool, causes []metav1.StatusCause) {
 	// used to validate uniqueness of boot orders among disks and interfaces
 	bootOrderMap = make(map[uint]bool)
+	// to perform as set of volume / fs names
+	diskAndFilesystemNames := make(map[string]interface{})
```

Comment:
nit: I think it's somewhat more idiomatic/common to use a `map[string]struct{}`, and then "signal the presence" of a disk/fs by having a `diskAndFilesystemNames[disk.Name] = struct{}{}`.

Reply:
Cool. I'm still new to Go, so good to know :)

Reply:
Here's a related post by the prophet Dave Cheney:
https://dave.cheney.net/2014/03/25/the-empty-struct